themes/src/main/resources/theme/keycloak-preview/account/resources/styles.css

```diff
@@ -301,11 +301,35 @@ p.description {
   position: absolute;
   right: 10px;
 }
+.list-view-pf-additional-info .fa-user{
+  margin-right: 5px;
+}
+.linked-account-list .list-view-pf .list-group-item-heading {
+    -ms-flex: 1 0 calc(40% - 20px);
+    flex: 1 0 calc(40% - 20px);
+  }
+.linked-account-list .list-view-pf-additional-info{
+    width:10%
+  }
+.linked-account-list .list-view-pf .list-group-item-text {
+    width: calc(45% - 40px);
+  }
 
 @media (max-width: 767px) {
   .card-pf-application h2 {
     margin-top: -10px;
   }
+  .card-linked-account .card-pf-body{
+    padding-left: 0;
+    padding-right: 0;
+  }
+  .card-linked-account .list-group-item{
+    padding: 10px 0px;
```

Comment:
1, need to add space between the selector and curly braces
2, remove the "px", avoid specifying units for zero values